src/common/WindowsTypes.hpp

```diff
@@ -0,0 +1,41 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-24, Lawrence Livermore National Security, LLC
+// and RAJA Performance Suite project contributors.
+// See the RAJAPerf/LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+///
+/// Type and value definitions to support Windows builds.
+/// Windows has different syntax and does not support all POSIX modes
+/// that Linux systems do.
+///
+
+#ifndef RAJAPerf_WindowsTypes_HPP
+#define RAJAPerf_WindowsTypes_HPP
+
+#include<sys/types.h>
```

Comment:
Nitpick, but it would be nice to have a space between `#include` and the file name.

Reply:
We currently do both (space and no space) throughout the code. After I get the azure and GH Actions CI PRs in, I will attempt to set up a style check action. That should be a good way to make things like this consistent.